src/components/va-back-to-top/va-back-to-top.css

```diff
@@ -0,0 +1,101 @@
+@import '../../../node_modules/animate.css/source/sliding_entrances/slideInUp.css';
+@import '../../../node_modules/animate.css/source/sliding_exits/slideOutDown.css';
+@import '../../mixins/buttons.css';
+:host {
```

Comment:
This is a _lot_ of CSS for this little button. Did you copy it from somewhere, or craft it by hand?

If copied, is there anything we can take out?

Reply:
I've taken a bit more out, but at this point I think it's as minimal as it can get.